hedera-node/src/main/java/com/hedera/services/store/models/Account.java

```diff
@@ -92,13 +105,52 @@ public void associateWith(List<Token> tokens, int maxAllowed) {
 		associatedTokens.addAllIds(uniqueIds);
 	}
 
-	public void dissociateWith(List<Token> tokens) {
+	/**
+	 * Performs validation and dissociation - removing relationships between tokens and accounts.
+	 * Expired tokens are transferred back to the treasury
```

Comment:
We have to clarify that: `Tokens of type FUNGIBLE_COMMON if expired get transferred back to the treasury`